base_layer/transactions/src/consensus.rs

```diff
@@ -56,4 +67,10 @@ impl ConsensusRules {
     pub fn emission_schedule(&self) -> &EmissionSchedule {
         &self.emission_schedule
     }
+
+    /// This returns the FTL(Future Time Limit) for blocks
+    /// Any block with a timestamp greater than this is rejected.
+    pub fn ftl(&self) -> DateTime<Utc> {
```

Comment:
```suggestion
    pub fn max_allowed_timestamp(&self) -> DateTime<Utc> {
```

Reply:
I would prefer to keep it FTL as that is the  known "industry" term